src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/copyQuestionModal.tag

```diff
@@ -13,16 +15,16 @@
                 </button>
                 <h4 class="modal-title" id="copyModalTitle">Copy Questions</h4>
             </div>
-            <div class="modal-body padding-0">
+            <div class="modal-body">
                 <form class="form" id="copyModalForm" role="form" method="post"
                     action="<%= Const.ActionURIs.INSTRUCTOR_FEEDBACK_QUESTION_COPY %>">
-                    <%-- Previous Questions --%>
+                    
                     <table class="table-responsive table table-hover table-bordered margin-0" id="copyTableModal">
                         <thead class="fill-primary">
                             <tr>
                                 <th style="width:30px;">&nbsp;</th>
                                 <th onclick="toggleSort(this);" id="button_sortid" class="button-sort-ascending"> 
-                                    Course ID <span class="icon-sort sorted-ascending"></span>
+                                    Course <span class="icon-sort sorted-ascending"></span>
```

Comment:
As a workaround to the unwanted wrapping of the sort icon, maybe we could add `style="width:100px"` for now, until we find a better way to solve it